modules/lvms-about-lvmcluster-cr.adoc

```diff
@@ -101,7 +101,16 @@ If any of these conditions are true, do not force wipe the disk. Instead, you mu
 
 |`deviceClasses.thinPoolConfig`
 |`object`
-|Contains the configuration to create a thin pool in the LVM volume group.
+|Contains the configuration to create a thin pool in the LVM volume group. 
+
+If you exclude this field, logical volumes are thick-provisioned. 
```

Comment:
```suggestion
If you exclude this field, the logical volumes are thick provisioned. 
```
I don't think the hyphen is needed in this instance since the words are acting as a complement to a noun rather than an adjective or adverb directly proceeding the noun.